### ISSUE TITLE
Turnout improvements

### ISSUE THREAD

**Issue by davidcutting42** [DCC-EX/CommandStation-EX#82]
- Turnout processing/Better class (may need a filter to handle WiThrottle integration)
 - Turnout names? ex: "coal yard"
 - Extended turnout command to handle servo/stepper angle?

**Comment by matt-hu** [comment]
If turnouts are being improved, a way of mapping turnouts to IO pins might be useful...

**Comment by Asbelos** [comment]
Yes tpl has this already. On 21 Nov 2020 17:53, Matt <notifications@github.com> wrote:
If turnouts are being improved, a way of mapping turnouts to IO pins might be useful...
